content/tutorials/single-page-tutorials/proof-inclusion-of-a-block.md

```diff
@@ -0,0 +1,351 @@
+      consoleColor,
+      'Block successfully notarized and stored at:'
+    );
+    console.log(filePath, '\n');
+
+    console.log(
+      consoleColor,
+      'The Notarized block can now be handed over to the verifier',
+    );
+  }
+}
+```
+
+#### 3. Get notarization
+As described in the previous step, the `getNotarization()` function is called from within `main()`. The function fetches the metadata of a given blockId 10 times while pausing for 1 second after each try via the `sleep()` function. As soon as a block was referenced by a milestone, the notarization for the block is fetched from the proof of inclusion plugin and  returned by the function. In case the block was not referenced by a milestone after 10 seconds, the function returns `false`.
+
+```javascript
+// Check for block confirmation and return proof of inclusion, if confirmed after n tries
+async function getNotarization(client, nodeURL, blockId) {
+  try {
+    console.log(
+      consoleColor,
+      'Wait for milestone confirmation to get notarized block:',
+    );
+
+    let i = 0;
+    while (i < 10) {
+      i++;
+      // Function waits for a certain time between iterations
+      await sleep(1000);
+
+      const blockMetadata = await client.blockMetadata(blockId);
+
+      // If a block was referenced by a milestone, the metadata contains the field 'referencedByMilestoneIndex'
+      if ('referencedByMilestoneIndex' in blockMetadata) {
+        console.log(
+          `Try ${i}: Block was referenced by milestone #${blockMetadata.referencedByMilestoneIndex}`,
+          '\n',
+        );
+
+        // Call "create" endpoint of PoI plugin with blockId and return the result
+        const poiPluginUrl = `${nodeURL}/api/poi/v1/create/${blockId}`;
+        const response = await fetch(poiPluginUrl);
+        const result = await response.json();
+
+        return result;
+
+      } else {
+        console.log(`Try ${i}: Block was not yet referenced by a milestone`);
+      }
+    }
+    console.log(`Block was not referenced by a milestone after ${i} seconds.`);
+
+    return false;
+
+  } catch (error) {
+    console.log(error);
+  }
+}
+```
+
+#### 4. Delay function
+The function `sleep()` receives a number of milliseconds and acts as a time delay for whichever function is calling it.
+
+```javascript
+function sleep(ms) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+```
+
+#### 5. Execute main function
+As a final step the function `main()` is called in order to trigger the process as described above.
+
+```javascript
+main().catch((err) => console.error(err));
+```
+
+---
+
+### Verify Notarization Script
+
+Create a new file named `verify-notarization.js`. As with the previous script (`create-notarization.js`), the code below is separated into distinct code snippets together with an explanation. After you have followed along the different steps, you can copy all snippets into the newly created file, following their respective order.
```

Comment:
```suggestion
Create a new file named `verify-notarization.js`.

Just like with the [Create Notarization](#create-notarization-script) script, we broke the code into separate snippets. To make it work, copy all code snippets one after another into the file that you have just created.
```

Similar to the previous suggestion. I dropped "to help you understand it better" to make space for the reference to the previous script. This bit is not crucial and the reader will probably recall it anyway.